include/onnxruntime/core/framework/run_options.h

```diff
@@ -40,6 +48,8 @@ struct OrtRunOptions {
   // /include/onnxruntime/core/session/onnxruntime_run_options_config_keys.h
   onnxruntime::ConfigOptions config_options;
 
+  onnxruntime::InlinedVector<const onnxruntime::lora::LoraAdapter*> active_adapters_;
```

Comment:
nit: struct data member naming

https://google.github.io/styleguide/cppguide.html#Variable_Names

```suggestion
  onnxruntime::InlinedVector<const onnxruntime::lora::LoraAdapter*> active_adapters;
```